dev/make-distribution.sh

```diff
@@ -201,6 +201,12 @@ fi
 # Copy data files
 cp -r "$SPARK_HOME/data" "$DISTDIR"
 
+# Make pip package
+echo "Building python distribution package"
+cd $SPARK_HOME/python
+python setup.py sdist
```

Comment:
Speed might not be a problem as you said. My concern is this step might need the users to meet some requirements they don't expect and don't want to use, e.g., some python modules.